common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -772,6 +777,12 @@ public void onReceive(@NonNull PropertyBag dataBag) {
 
                             LocalBroadcaster.INSTANCE.unregisterCallback(RETURN_AUTHORIZATION_REQUEST_RESULT);
 
+                            // If we received an exception during the receiver callback execution,
+                            // we should set that as the command result
+                            if (receiverException[0] != null) {
+                                commandResult = CommandResult.of(CommandResult.ResultStatus.ERROR, receiverException, correlationId);
```

Comment:
I think we should pass in `receiverException[0]` to this method

```suggestion
                                commandResult = CommandResult.of(CommandResult.ResultStatus.ERROR, receiverException[0], correlationId);
```